components/Wizard/ImageStep.js

```diff
@@ -567,6 +567,198 @@ class ImageStep extends React.PureComponent {
       </>
     );
 
+    const ostreeContainerFields = (
```

Comment:
Should this not be the same as the `ostreeCommitFields`, how come it was split?

Reply:
Ah yes. I split it thinking the UI for image types would be different. Removing the duplicate code now.

Reply:
They should be the same in this case (whether the code is the same is of course up to you, just wanted to verify that the UX is identical).